python/TestHarness/testers/Tester.py

```diff
@@ -571,7 +571,27 @@ def checkRunnableBase(self, options):
         """
         reasons = {}
         checks = options._checks
-        capabilities = options._capabilities
+
+        # augment capabilities of the application with specs of the current tester
```

Comment:
Can we make this static of sorts so that we can unit test it with how we plan on using it